api/src/main/java/io/druid/data/input/impl/TimestampSpec.java

```diff
@@ -17,14 +17,14 @@
 
 package io.druid.data.input.impl;
 
+import java.util.Map;
```

Comment:
This import ordering is a little off from normal style, if you're using IntelliJ then https://github.com/druid-io/druid/raw/master/intellij_formatting.jar should adjust it automatically